.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Ensure that environment variables like GITHUB_TOKEN and OPENAI_API_KEY are handled securely and not hard-coded or exposed in version control. Consider using a .env file and a library like dotenv to load them securely in your application.